pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -1322,7 +1322,7 @@ public void createSubscription(
                     defaultValue = "latest",
                     allowableValues = "latest, earliest, ledgerId:entryId"
             )
-                    MessageIdImpl messageId,
+                    ResetCursorData resetCursorData,
```

Comment:
Oh, Sorry @RobertIndie my mistake, it looks like we already have `messageId` there, changing to resetCursorData will introduce compatibility issues.

Reply:
This is the REST API. The Json format of the `ResetCursorData` and `MessageIdImpl` used here is the same. It looks like there are no compatibility issues here.